src/assets/SvgIcons/LocationRefresh.tsx

```
@@ -0,0 +1,22 @@
+import SvgIcon from "src/components/shared/SvgIcon";
+
+const LocationRefresh = (props: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)